### PR TITLE
doc: Fix doc for mds cap acquisition throttle

### PR DESCRIPTION
**Pull request by kotreshhr**
The config values picks the definition from the
description. Removed the duplicate description
in the doc.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by kotreshhr** [comment]
Existing Documentation with duplicate description - https://docs.ceph.com/en/latest/cephfs/cache-configuration/#mds-cap-acquisition-throttle

File: src/common/options/mds.yaml.in [hunk 1]
```diff
@@ -347,22 +347,22 @@ options:
 - name: mds_session_cap_acquisition_throttle
   type: uint
   level: advanced
-  desc: throttle point for cap acquisition decay counter
+  desc: The throttle point for cap acquisition decay counter
```
Comment (anthonyeleven) [inline src/common/options/mds.yaml.in:350]: Suggest `Threshold at which the cap acquisition decay counter throttles`

**Review by anthonyeleven — APPROVED** [review]
Non-blocking suggestion entered

File: src/common/options/mds.yaml.in (hunk 1 above)
Comment (kotreshhr) [reply src/common/options/mds.yaml.in]: Done.

File: src/common/options/mds.yaml.in [hunk 2]
```diff
@@ -347,22 +347,22 @@ options:
 - name: mds_session_cap_acquisition_throttle
   type: uint
   level: advanced
-  desc: throttle point for cap acquisition decay counter
+  desc: Threshold at which the cap acquisition decay counter throttles
```
Comment (batrick) [inline src/common/options/mds.yaml.in:350]: convention is that "desc" is not a sentence and the first word is not capitalized.

File: src/common/options/mds.yaml.in [hunk 3]
```diff
@@ -347,22 +347,22 @@ options:
...
   default: 500000
   services:
   - mds
 - name: mds_session_max_caps_throttle_ratio
   type: float
   level: advanced
-  desc: ratio of mds_max_caps_per_client that client must exceed before readdir may
+  desc: The ratio of mds_max_caps_per_client that client must exceed before readdir may
```
Comment (batrick) [inline src/common/options/mds.yaml.in:357]: article adjectives also not conventionally required for "desc"

**Review by batrick — CHANGES_REQUESTED** [review]

File: src/common/options/mds.yaml.in (hunk 2 above)
Comment (kotreshhr) [reply src/common/options/mds.yaml.in:350]: ok, I can change that. But this gets imported to documentation where it becomes incorrect. Is that fine?

**Comment by joscollin** [comment]
jenkins render docs

File: src/common/options/mds.yaml.in (hunk 2 above)
Comment (zdover23) [reply src/common/options/mds.yaml.in:350]: @kotreshhr, @batrick: I'll do my best to institutionalize this convention of making "desc"s neither sentences nor capitalized.

**Comment by batrick** [comment]
@kotreshhr what's blocking an update?

**Comment by kotreshhr** [comment]
> @kotreshhr what's blocking an update?

@batrick sorry had missed this. Updated.

**Review by batrick — APPROVED** [review]

**Comment by batrick** [comment]
jenkins test make check

**Comment by kotreshhr** [comment]
jenkins test make check

**Comment by kotreshhr** [comment]
jenkins test make check arm64

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/53024 - Reef backport
https://github.com/ceph/ceph/pull/53025 - Quincy backport

**Pull request merged**